app/src/main/java/com/duckduckgo/app/browser/WebViewLongPressHandler.kt

```diff
@@ -40,11 +40,11 @@ interface LongPressHandler {
 
 class WebViewLongPressHandler @Inject constructor() : LongPressHandler {
 
-    override fun handleLongPress(longPressTargetType: Int, longPressTargetUrl: String, menu: ContextMenu) {
+    override fun handleLongPress(longPressTargetType: Int, longPressTargetUrl: String?, menu: ContextMenu) {
         when (longPressTargetType) {
             WebView.HitTestResult.IMAGE_TYPE,
             WebView.HitTestResult.SRC_IMAGE_ANCHOR_TYPE -> {
-                if(URLUtil.isNetworkUrl(longPressTargetUrl)) {
+                if(URLUtil.isNetworkUrl(longPressTargetUrl) || URLUtil.isDataUrl(longPressTargetUrl)) {
```

Comment:
Needs space after `if` (didn't used to bother flagging these but I can see changes elsewhere suggest `if (` is the format we're going after.